polyglot.js

```diff
@@ -2,27 +2,56 @@ class PolyGlot {
+				}
+		}
+
+		static get languages() {
+			switch(game.system.id.toLowerCase()){
+				default:
+					return CONFIG[game.system.id.toUpperCase()].languages;
+				case "wfrp4e":
+					let re = /Language \((.+)\)/i;
+					const pack = game.packs.get("wfrp4e.skills");
+					return pack.getIndex().then( itemList => {
+						let langs = {};
+						for (let item of itemList){
+							let found = item.name.match(re);
+							if(found && found.length > 1 && found[1].length >= 1){
```

Comment:
Just checkout that `found` is not null is enough, the length checks for the array is always be > 1 as well as for the first group (since it's a mandatory capture group with a minimum of one character).
Also, found isn't modified, so it should be `const`
also, the regular expression isn't reused in multiple places, so I'd prefer if it stays with the code that actually uses it. No need to separate it into a variable that is set somewhere else.
Personal preference: I don't like having `{ }` braces for one-line if statements. Also, missing space between `)` and `{`
I would just do : 
```js
const language = item.name.match(/Language \((.+)\)/i)
if (language)
   langs[language[1]] = language[1];
```

Reply:
Your preference for 1-liners to not have { } seems to be at odds with your desire to always have a break; statement in a switch statement :)   I tend to always add brackets regardless cause people make changes later on that break it.  But happy to change it to what your style is

Reply:
Yep, I'm aware of that, it's definitely one thing that people say should always be specified (the `{ }`) to avoid the risk of error in changes, but my brain just doesn't like seeing those one liners with the braces. So I know it goes against the rest of my coding style, but I really like the clean look of not having braces unless required.